pallets/xcm-helper/src/lib.rs

```diff
@@ -517,27 +460,27 @@ pub mod pallet {
 		}
 	}
 
-	impl<T: Config> Convert<MultiLocation, Option<u128>> for Pallet<T> {
-		fn convert(_a: MultiLocation) -> Option<u128> {
-			todo!()
-		}
-	}
-
 	impl<T: Config> TransactAsset for Pallet<T> {
 		/// Generate Ingress Message for new Deposit
 		fn deposit_asset(what: &MultiAsset, who: &MultiLocation) -> Result {
-			<IngressMessages<T>>::try_mutate(|ingress_messages| {
-				ingress_messages.try_push(TheaMessage::AssetDeposited(
-					Box::new(who.clone()),
-					Box::new(what.clone()),
-				))
-			})
-			.map_err(|_| XcmError::Trap(10))?;
 			Self::deposit_event(Event::<T>::AssetDeposited(who.clone(), Box::new(what.clone())));
+			// Create approved deposit
+			let MultiAsset { id, fun } = what;
+			let who =
+				T::AccountIdConvert::convert_ref(who).map_err(|_| XcmError::FailedToDecode)?;
+			let amount: u128 = Self::get_amount(fun).ok_or(XcmError::Trap(101))?;
```

Comment:
Sorry, I just wondering what 101 means. I think such status codes should be moved to the constants with proper names and description, otherwise - how consumer supposed to know what 101 means...

Reply:
I created [issue](https://github.com/Polkadex-Substrate/thea/issues/194) for this. I will fix this later.